docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
+a web browser to communicate with your server securely over an encrypted SSL (secure socket layer)
+connection. The certificate also identifies your website so that the browser (and the person behind it)
+can trust your identity. SSL certificates have mainly been used to provide secure,
+encrypted communication for sensitive information like financial and health data, but
+we now recommend all websites run on SSL to prevent eavesdropping and content hijacking.
+
+You can use Certbot to easily obtain and configure a free certificate from Let's Encrypt, a
+joint project of EFF, Mozilla, and many other sponsors.
+
+Certificates and Lineages
+=========================
+
+Certbot introduces the concept of a *lineage,* which is a collection of all the versions of a certificate
+plus additional information that Certbot uses to maintain the configuration of your webserver from
+renewal to renewal. Whenever you renew a certificate, Certbot keeps the same configuration unless
+you explicitly change it. If you add domains, you can either add them to an existing lineage or create
+a new one. 
+
```

Comment:
First, let me share the reaction of a person who was learning certbot from scratch:  "lineage" is a rather confusing term.
(I saw it being discussed in one of the recent threads here, and I agree.)  It is _sort of_ understandable, but I would have used "set", "family", or, possibly wouldn't not introduce it at all. I would rather think of different versions of the same certificate, calling it collectively a "certificate".
(For comparison: you don't call the collection of different versions of the same file in the source control a _lineage_, right?) 


My biggest confusion is what is the difference between _`lineage`_ and `Certificate Name`. 
Can someone please explain the [practical] difference? So far, from the practical point of view, _Certificate Name_ is used to label the _lineage_ (and its components). If you are intending to keep "lineage" and describe it, I would suggest incorporating that explanation in this document.

Second, I think including the "additional information that Certbot uses to maintain the configuration of your webserver from renewal to renewal" into the notion of lineage is confusing. It is like claiming that birth certificates and county records of birth, marriage and death are a part of your family. I would call those lineage (or, more exactly certificate) configuration.

Yet another suggestion: While a big focus of certbot (and LetsEncrypt) is at the certificates for websites, these certificates are also used for other servers (IMAP/POP3, MTA/MSA, ... ). I think the documentation should be talking more generally about "servers" (as opposed to webservers).